### PR TITLE
First version of the waterbodies docker image

### PR DESCRIPTION
**Pull request by vikineema**

### CODE REVIEW

File: waterbodies/db_models.py
Comment (caitlinadams) [inline waterbodies/db_models.py:1]: Noting that the TODO to change waterbody_observations_2 to waterbody_observations will require an updated schema, as we're now including task_id in the schema.

File: waterbodies/surface_area_change.py
Comment (caitlinadams) [inline waterbodies/surface_area_change.py:1]: I've updated this file to make the logic for getting the pixel counts a bit cleaner.

File: README.md
Comment (caitlinadams) [inline README.md:1]: It would be nice to update this with some basic instructions before merging into main

**Review by caitlinadams — APPROVED** [review]
Thanks for this PR, @vikineema -- it's great to see the code coming together.

I made some changes to how the masking and pixel counts work. Could you re-run your process tests and overwrite the current results in the waterbodies_observations_2 table? We can then compare this to the original waterbodies_observations table that we still have in the database.

File: waterbodies/db_models.py
Comment (vikineema) [reply waterbodies/db_models.py:1]: I've updated the schema using alembic

**Pull request merged**